AppProgrammingQuickest-start.md

```diff
@@ -0,0 +1,25 @@
+# App Programming Quickest-start
+
+This shows you how to add code into a file, add it to main menu and run it.
+See https://github.com/emfcamp/tidal-docs/blob/main/AppQuickstart.md for more example code!
+
+1. Plug in your TiDAL in your computer with power off
+2. Turn on your TiDAL
+3. Open https://editor.badge.emfcamp.org/ in Google Chrome
+4. Press "Connect", select your TiDAL (Espressif CDC Device on Windows) and press "Connect"
+5. Press "Programming & Files"
+6. Press "Make folder", second button from the left, and create a folder named `apps`
```

Comment:
This step is no longer needed (with v0.5.1 or higher firmware)

Suggest we add a "check your firmware is up to date" step at the beginning?